index.html

```diff
@@ -11,7 +11,75 @@
     <title>TripHouse_site</title>
   </head>
   <body>
-    <div class="top-section"></div>
+    <div class="top-section">
+      <div class="container">
+        <div class="top-frame">
+          <img src="./images/svg/logo_vector.svg" alt="logo" class="top-logo" />
+          <div class="top-group">
+            <p class="top-stays hovered">Stays</p>
+            <p class="top-attractions hovered">Attractions</p>
```

Comment:
Try  not to use proper names in names of the classes (stays, attractions, NewYork, Mickey-Mouse, etc.). You code structure will be live more longer, than content &#128521;